mana_decay_provider.go

```diff
@@ -1,6 +1,9 @@
 package iotago
 
 import (
+	"fmt"
+	"math"
```

Comment:
this package is used just for the function I'm supposed to move to the test file